### DISCUSSION TITLE
Output to WPF TextBox Example

### DISCUSSION THREAD

**Discussion by shaggygi** [Tyrrrz/CliWrap#141] [Q&A]
Very cool API/project. 👍 

Do you happen to have any examples showing how to hook up and output to a WPF TextBox?

Any info is appreciated.

**Comment by Sadness96** [comment]
Hello, I just want to use CliWrap to do some functions, Encountered some difficulties and couldn't stop,but it can be displayed in WPF
![image](https://user-images.githubusercontent.com/19670054/164162532-5d3fcee8-dee0-4c6d-93d6-fdbe8ec847d9.png)
[https://github.com/Sadness96/EasyDeploy](https://github.com/Sadness96/EasyDeploy)

**Reply by Tyrrrz** [reply]
If you have a property on the view model such as `string Text { get; set; }` then it should be as straight-foward as `WithStandardOutputPipe(PipeTarget.ToDelegate(s => Text += Environment.NewLine + s))`
